sdks/python/apache_beam/runners/interactive/dataproc/dataproc_cluster_manager.py

```diff
@@ -65,13 +66,18 @@ def __init__(
           cluster_name)
       self._cluster_name = cluster_name
     else:
-      self._cluster_name = self.DEFAULT_NAME
+      self._cluster_name = ie.current_env().clusters.default_cluster_name
```

Comment:
Based on a few recent suggestions, I have now made it so that the DataprocClusterManager takes in an instance of MasterURLIdentifier, which has all of the attributes necessary for cluster configuration (project_id, region, cluster_name). If the cluster_metadata.cluster_name is empty, we default it here, otherwise we have noop. With this, do you think it would still be necessary to add some `custom_cluster=False` parameter in the constructor?